README.md

```diff
@@ -1 +1,73 @@
-# Agorex - Cooperative Decentralized Exchange
\ No newline at end of file
+# Agorex - The Cooperative Decentralized Exchange
+[![Chat on Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/GVuj6UY)
+[![Join us on Reddit](https://img.shields.io/badge/reddit-Agorex-red.svg)](https://www.reddit.com/r/Agorex/)
+
+Our official mission statement is currently pending. 
+
+This statement, along with the goals, [future](#milestones), and [members](#members) of Agorex are to be determined by the Agorex cooperative.  
+
+- [Name](#name)
+- [Contribute](#contribute)
+- [Milestones](#milestones)
+- [Members](#members)
+- [Social Links](#social-links)
+
+## Name
+**Agorex, the Cooperative Decentralized Exchange**
+
+> So, what exactly does "Agorex" mean?
+
+Agorex is a combination of two words: _Agora_ & _Exchange_.
+
+An [_Agora_](https://en.wikipedia.org/wiki/Agora) was a _"public, open space used for assemblies and markets"_ in ancient Greece.
+
+As an added bonus _Agora_ also means _"Now"_ in Portuguese!
+
+We believe this name represents our core values of providing a **public**, **open**, **honest**, **community-run decentralized exchange**.
+
+
+## Contribute
+We are a group who have worked on a decentralized cryptocurrency exchange before. We want the community, like the technology, to also be decentralized. So we are building a community-run decentralized cryptocurrency exchange that puts the community in charge. To see how we put the community in charge of the direction of the exchange, see our proposal process. We also plan to set up a cooperative organization where contributors who want to make a more significant contribution can literally operate and manage the exchange on behalf of the community.
```

Comment:
Changes:

"who have experience with decentralized exchanges."

"So, we are building"

Remove the "To see how" sentence and talk about the proposal process while linking to the actual file.

Last sentence should be simpler: "We are in the final phases of setting up a cooperative so that any member can have an equal voice in decisions about the exchange."